### ISSUE TITLE
Ignoring paths inside arrays

### ISSUE THREAD

**Issue by drakulavich** [lukas-krecan/JsonUnit#107]
First of all, thanks for the great library!

I faced issue when comparing two documents with structure like this:
Json 1
```
{
  "calendars": [
    {
      "dates": [
        {
          "id": "date-8c9c9ffa58fee47b47c2c1eb511c4e6c",
          "type": "NOTIFICATION",
          "from": 1525006134975
        }
      ]
    }
  ]
}
```

Json 2
```
{
  "calendars": [
    {
      "dates": [
        {
          "id": "date-8c9c9ffa58fee47b47c2c1eb511c4e6c",
          "type": "NOTIFICATION",
          "from": 1525008354975
        }
      ]
    }
  ]
}
```

My goal is to compare two jsons ignoring `from` field.
```
assertJsonEquals(expected, actual,
                whenIgnoringPaths(
                        "calendars[*].dates[*].from"
                ).when(IGNORING_ARRAY_ORDER)
        );
```
This code returns error:
```
java.lang.AssertionError: JSON documents are different:
Different value found when comparing expected array element calendars[0] to actual element calendars[0].
Different value found when comparing expected array element calendars[0].dates[0] to actual element calendars[0].dates[0].
```
 Looks like path is not ignored. If I change ignoring path to "from" it will work.

**Comment by lukas-krecan** [comment]
Hi, thanks. I will take a look at it next week. 

30. 4. 2018 14:14, 14:14, Anton Yakutovich <notifications@github.com> napsal/a:

**Comment by lukas-krecan** [comment]
Should be fixed i 1.29.0
